Documentation/botr/ryujit-overview.md

```diff
@@ -568,36 +590,31 @@ Full instructions for dumping the compilation of some managed code can be found
 It takes some time to learn to “read” the expression trees, which are printed with the children indented from the parent, and, for binary operators, with the first operand below the parent and the second operand above.
```

Comment:
> for binary operators, with the first operand below the parent and the second operand above.

I think this is not correct anymore.

Reply:
Thanks, missed this one.

Reply:
Deleted the entire thing. Stating obvious things such as "children indented from the parent" doesn't help.